packages/e2e/docker-compose.yml

```
@@ -0,0 +1,127 @@
+version: "3.3"
+
+services:
+  local-testnet:
+    build:
+      context: ./local-network
+    ports:
+      - 3001:3001
+    volumes:
+      - ${PWD}/local-network/sockets:/root/sockets
+      - ./local-network/config/network/${NETWORK:-mainnet}:/config/nodes # Mount the config so we can update the genesis files and hashes.
```

```suggestion
      - ./local-network/config:/config/nodes # Mount the config so we can update the genesis files and hashes.
```
Is there a reason to support multiple network configurations?

---

Not needed since we are using our custom config